test/unit/llm-events/error.test.js

```diff
@@ -22,6 +23,12 @@ tap.test('LlmErrorMessage', (t) => {
     'vector_store_id': undefined,
     'tool_id': undefined
   }
-  t.same(errorMsg, expected)
-  t.end()
+  assert.equal(errorMsg['http.statusCode'], expected['http.statusCode'])
+  assert.equal(errorMsg['error.message'], expected['error.message'])
+  assert.equal(errorMsg['error.code'], expected['error.code'])
+  assert.equal(errorMsg['error.param'], expected['error.param'])
+  assert.equal(errorMsg.completion_id, expected.completion_id)
+  assert.equal(errorMsg.embedding_id, expected.embedding_id)
+  assert.equal(errorMsg.vector_store_id, expected.vector_store_id)
+  assert.equal(errorMsg.tool_id, expected.tool_id)
```

Comment:
I think when migrating from `t.same` to individual assertions, the `get [Symbol.toStringTag]()` getter isn't getting called. you should add an assertion of `errorMsg.toString()`

Reply:
Ok, is `assert.ok(errorMsg.toString())` sufficient?

Reply:
i would `assert.equal` the actual value

Reply:
For what it's worth, in the suite I'm currently converting I have added a utility to replace `t.same`. Once I can get some code committed, I'll back up and add the utility individually for all of to use.